README.md

```diff
@@ -1,3 +1,167 @@
 [![Travis](https://img.shields.io/travis/triplea-game/dice-server-js.svg?style=flat-square)](https://travis-ci.org/triplea-game/dice-server-js)
 # dice-server-js
-The new TripleA dice server powered by Node.js
+The new TripleA dice server re-written in JavaScript.
+
+## Setup
+### Prequisites
+In order to be able to run the server you need to have a couple of things installed on your system:
+ - `node.js` >= 10
+ - `yarn`
+ - `postgres` >= 10
+After those are installed on your system you can follow the steps below to get yourself up and running.
+The steps assume node and yarn are on the `PATH` of your system, if that's not the case, you need to replace `node` and `yarn` with the full qualified path name to the executable.
+
+### One-Time-Setup
+1. Clone this Repository in a directory of your choice.
+2. Open a terminal and navigate to the newly checked out directory.
+3. Run `yarn install`, this will resolve all dependencies for you and install them in `node_modules`
+4. - Generate an RSA key-pair, if you have installed `openssl` and it's on your PATH, you can simply run `yarn generate-keypair`, which will create the files `privkey.pem` and `pubkey.pem` in the current directory.
+   - On productive systems you should move them into a "save directory" and (at least on UNIX-like systems) restrict read access of the private key to the user that's going to run the server and root.
+5. - Create a file `config.json` which contains all information the server needs to start.
+   - Alternatively those configuration options can be passed via CLI directly or as environment variables, check the [nconf docs](https://github.com/indexzero/nconf#example) for more information.
+   - The config files' Layout will be explained below.
+6. Create a Database in postgres to be used by the Server.
```

Comment:
```suggestion
6. Create a database in Postgres to be used by the server.
```

I noted both "Database" and "database" used throughout the docs.  I'm assuming that's an artifact from German nouns. :smile:  I think "database" would be more appropriate here, but using a consistent capitalization is probably more important so as not to confuse a reader who may think there's something special about "The Database" vs. "a database."

Ditto for "Server" vs. "server".

Reply:
Yeah, it's a real struggle every time.
When I actively think about it it's written correctly: `database`.
But when I'm just typing a lot `Database` is simply more intuitive for me 😅

Reply:
Will do, but it will probably take a couple of hours: I want to finish all the tests before that.